### DISCUSSION TITLE
Maintenance branch | hot fix release

### DISCUSSION THREAD

**Discussion by Kickbuttowaski** [semantic-release/semantic-release#3557] [Configuration]
Hi,

I am using the semantic release library to publish my react project.

The current version is v1.22.3. and I want to make a fix over the v1.21.1

I tried creating a branch named 1.21.8 with the changes and pushed, but was getting the below error

  _Completed step "fail" of plugin "@semantic-release/github"
19:22:04 [1:52:04 PM] [semantic-release] › ✘  EINVALIDNEXTVERSION The release `1.21.2` on branch `1.21.8` cannot be published as it is out of range.
19:22:04 Based on the releases published on other branches, only versions within the range >=1.21.1 <1.21.2 can be published from branch 1.21.8._

How can I do this using maintenance branch? Tried going through the documentation and it's kind of confusing
They have mentioned like either pushing or merging to the maintenance branch, not sure which one to pick. Can someone help on this?
https://semantic-release.gitbook.io/semantic-release/usage/workflow-configuration#pushing-to-a-maintenance-branch

I am using the below config

    branches: [
        { name: 'poc', channel: 'latest' },
        { name: 'stage', channel: 'latest' },
        { name: 'main', channel: 'latest' },
        { name: '1.21.8', range: "1.21.x", channel: "1.21.x" },
      ]

**Comment by travi** [comment]
What is the meaning of the .8 in your branch name? Normally, it would be best to name that branch 1.21.x

**Reply by Kickbuttowaski** [reply]
@travi , just for testing purpose have hard coded the branch name.
Does it have any impact while generating the build?
